casepro/cases/views.py

```diff
@@ -390,6 +391,14 @@ def post(self, request, *args, **kwargs):
     class List(OrgPermsMixin, SmartListView):
         paginate_by = None
 
+        @staticmethod
+        def humanise_minutes(minutes):
+            if minutes < 60:
+                return "%.0fm" % ceil(minutes)
+            else:
+                hrs, mins = divmod(minutes, 60)
+                return "%.fh %.0fm" % (hrs, ceil(mins))
```

Comment:
Maybe we should also allow for days? I'm not sure what the expected average response times are supposed to be, but converting 112h -> 4d 16h is not really something that's easy to do in your head.